docs/organizations/managing-security-and-risk.md

```diff
@@ -223,7 +223,7 @@ To revoke the Security Manager role:
 
 ## Data retention
 
-Codacy retains all items indefinitely.
+Codacy retains all items indefinitely, except in the cases below.
```

Comment:
I would put it the other way around, to avoid readers retaining only the first part. 😉
WDYT?

```suggestion
Except for the situations described below, Codacy retains all items indefinitely.
```